docs/cinder-csi-plugin/sidecarcompatibility.md

```diff
@@ -11,6 +11,6 @@
 
 ## Set file type in provisioner
 
-There is a change in [csi-provisioner 2.0](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-2.0.md): The fstype on provisioned PVs no longer defaults to "ext4". A defaultFStype arg is added to the provisioner. Admins can also specify this fstype via storage class parameter. If fstype is set in storage class parameter, it will be used. The sidecar arg is only checked if fstype is not set in the SC param.
+There is a change in [CSI-provisioner 2.0](https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-2.0.md): The fsType on provisioned PVs no longer defaults to "ext4". A defaultFSType argument is added to the provisioner. Admins can also specify this fstype via storage class parameter. If fsType is set in storage class parameter, it will be used. The sidecar argument is only checked if fstype is not set in the SC param.
```

Comment:
I'd remove the dash between CSI-provisioner, seems like they go without in their README: https://github.com/kubernetes-csi/external-provisioner/tree/master.

Linked release note uses `fstype` as the name of the property, let's stick to that. `defaultFSType` is fine, but please use ticks to mark it as code.